### PR TITLE
Only confirm new subscriptions

### PR DESCRIPTION
**Pull request by cap10morgan**
turbovote-web users were seeing a confirmation on every step change because we're (unfortunately) re-PUT'ing the subscriptions on each one for now.

This makes subscription PUTs idempotent w/r/t confirmation messages.

### CODE REVIEW

File: src/election_notification_works/entities/subscription.clj [hunk 1]
```diff
@@ -15,28 +15,27 @@
...
-  (some-> (read-subscription-entity db user-id)
-          entity->subscription))
+  ([user-id] (read-subscription (db/db) user-id))
+  ([db user-id]
+   (some-> (read-subscription-entity db user-id)
+           entity->subscription)))
 
 (defn update-subscription
   [old-subscription new-subscription]
-  (if (empty (set/difference (:mediums old-subscription)
```
Comment (ericnormand) [inline src/election_notification_works/entities/subscription.clj]: Ah! I'm glad you got rid of this. Looks like a mistake between `empty` and `empty?`

Comment (cap10morgan) [inline src/election_notification_works/entities/subscription.clj]: Exactly. You'll see that it's now using `empty?` in its new home in the `handlers` ns.

Comment (ericnormand) [inline src/election_notification_works/entities/subscription.clj]: Yep!

**Comment by ericnormand** [comment]
Nice tests.

**Comment by ericnormand** [comment]
LGTM

**Comment by cap10morgan** [comment]
@ericnormand Thanks for the fast after-hours review!

**Pull request merged**